README.md

```diff
@@ -130,3 +130,18 @@ Overly complex structures and unnecessary code can be avoided by effective refac
 * Overly-nested and long functions
 * Long parameter lists
 
+
+## Code Reviews
+### [Evaluation Feedback process](https://smartbear.com/learn/code-review/guide-to-code-review-process/)
```

Comment:
Instead of linking the title add it at the bottom and advise employees to read more here, as it is quite easy to skip over the link without even realising it is a link

Reply:
done